knowledge-base/listview-kb-display-items-horizontally.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Displaying ListView Items Horizontally in Blazor
+description: Learn how to arrange the items in a TelerikListView for Blazor horizontally using custom CSS styles.
+type: how-to
+page_title: How to Arrange Telerik Blazor ListView Items Horizontally
+slug: listview-kb-display-items-horizontally
+tags: listview, blazor, layout, horizontal, css, styling
+res_type: kb
+ticketid: 1649286
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>ListView for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to display the items in the TelerikListView component horizontally.
+
+This KB article answers the following questions:
+- How to set the TelerikListView to horizontal?
+- Is it possible to display ListView items side by side in Blazor?
+
+## Solution
+
+To display the ListView items horizontally, apply the `display: inline-block;` CSS style to the `k-listview-item` HTML div element. This approach allows you to customize the appearance of each ListView item and arrange them side by side.
+
+`````CSHTML
+<style>
+    .horizontal-listview .k-listview-item {
+        height: 150px;
+        width: 150px;
+        display: inline-block;
+        margin: 10px;
+        border: 1px solid black;
+        border-radius: 10px;
+        padding: 10px;
+    }
+</style>
+
+<TelerikListView Data="@ListViewData"
+                 Class="horizontal-listview"
+                 Width="700px" 
+                 Pageable="true" 
+                 PageSize="4">
```

Comment:
Parameters like `Width` and `Height` can be put after more important ones that are related to data operations and features.

```suggestion
<TelerikListView Data="@ListViewData"
                 Class="horizontal-listview"
                 Pageable="true" 
                 PageSize="4"
                 Width="700px">
```